### ISSUE TITLE
Accountname clashes

### ISSUE THREAD

**Issue by xiffy** [gugray/rss-parrot#11]
Hi,
What a nice idea, RSS to Fediverse. I ran into an issue though. When a domain has more RSS Feeds there is only one account created. For instance I host some LastFM RSS feeds, all on the same domain: lfm.xiffy.nl 
When I added https://lfm.xiffy.nl/thexiffy and https://lfm.xiffy.nl/thexiffy/loved they got mingled together as one account, which seems incorrect.

**Comment by gugray** [comment]
The name of the accounts is derived from the name of the site, but it includes the full path. With the two feeds above, they both point back to https://lfm.xiffy.nl/ in their <link> tag at the top. In this case, the first added feed is meant to win.

I'll look into this to see if the two feeds do get mixed up. They should not; only the one that was added first should be retrieved for the accound.

**Comment by xiffy** [comment]
Ah off-course, link self.
My err :-) but this shows, more feeds could be on the wrong side. I'll fix my site.

**Comment by gugray** [comment]
Yup, this may well be an issue in existing data... I'll give it more thought.
